awe_lexica/setup/data.py

```diff
@@ -826,8 +831,11 @@ def to_disk():
         filename = "{variable_name}.json".format(
             variable_name=export
         )
-        with resources.path('awe_lexica.json_data',
-                            filename) as outputfile:
+        # with resources.path('awe_lexica.json_data',
+        #                     filename) as outputfile:
```

Comment:
Commented code should be removed unless it is crucial for understanding what the current code does. 

As a dev, I don't care that it used to use resources.path